tests/test_examples.py

```diff
@@ -27,20 +31,30 @@
 ]
 
 
-def get_file_paths():
-    """Gets all the files from the examples folder for testing. This will
+def get_file_paths(paths: Union[str, List[str]]):
+    """
+    Gets all the files from the examples folder for testing. This will
     return all the full file paths to the file, meaning they will have the
-    _EXAMPLES_PATH prefix followed by the file name for each element.
-    A list of all the file paths will be returned."""
+    path prefix followed by the file name for each element.
+    A list of all the file paths will be returned.
+
+    :param paths: The paths to retrieve the files from.
+    """
     test_files = []
-    for _, _, files in os.walk(_EXAMPLES_PATH, topdown=True):
-        for filename in files:
-            if filename not in IGNORED_TESTS and filename.endswith(".py"):
-                test_files.append(_EXAMPLES_PATH + filename)
+
+    if isinstance(paths, str):
+        paths = [paths]
+
+    for path in paths:
+        for _, _, files in os.walk(path, topdown=True):
+            for filename in files:
+                if filename not in IGNORED_TESTS and filename.endswith(".py"):
+                    test_files.append(path + filename)
+
     return test_files
 
 
-@pytest.fixture(scope="session", autouse=True)
+@pytest.fixture(scope="session")
 def symbols_by_file() -> Dict[str, Set[str]]:
```

Comment:
You can delete this function, and the corresponding pylint test case, entirely. For this task I'm asking you to replace the existing test case with one that instead runs PythonTA on the example files.